src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -166,19 +166,50 @@ _ray_indices(::Val{_cone_of_dim}, PF::Polymake.BigObject; c_dim::Int = 0) = Poly
 @doc raw"""
     cones(PF::PolyhedralFan)
 
-Return a list of all non-zero-dimensional cones of a polyhedral fan.
+Return the ray indices of all non-zero-dimensional
+cones in a polyhedral fan.
 
 # Examples
-The 12 edges of the 3-cube correspond to the 2-dimensional cones of its face fan:
 ```jldoctest
 julia> PF = face_fan(cube(3));
 
-julia> length(cones(PF))
-26
+julia> cones(PF)
+26×8 IncidenceMatrix
+[1, 3, 5, 7]
+[2, 4, 6, 8]
+[1, 2, 5, 6]
+[3, 4, 7, 8]
+[1, 2, 3, 4]
+[5, 6, 7, 8]
+[1, 5]
+[3, 7]
+[1, 3]
+[5, 7]
+[2, 6]
+[4, 8]
+[2, 4]
+[6, 8]
+[1, 2]
+[5, 6]
+[3, 4]
+[7, 8]
+[1]
+[5]
+[3]
+[7]
+[2]
+[6]
+[4]
+[8]
 ```
 """
 function cones(PF::_FanLikeType{T}) where T<:scalar_types
```

Comment:
```suggestion
function cones(PF::_FanLikeType)
```
`T` should be irrelevant here.

Reply:
Thx. I have rebased (and squashed) our changes and integrated this.